### PR TITLE
Drop compatibility with Plone 5.2 and lower.

### PR DESCRIPTION
**Pull request by mauritsvanrees**
Drop Python 2 compatibility.  Remove complete `_compat.py` file.
Use plone.base to fix deprecation warning.
I have updated coredev 5.2 to use new branch 1.x.

### CODE REVIEW

**Comment by mauritsvanrees** [comment]
@jenkins-plone-org please run jobs

File: setup.py [hunk 1]
```diff
@@ -12,22 +11,15 @@
...
     description="Replacement for isURLInPortal method in Plone",
     long_description=long_description,
     # Get more strings from https://pypi.org/classifiers/
     classifiers=[
         "Development Status :: 5 - Production/Stable",
         "Framework :: Plone",
-        "Framework :: Plone :: 4.3",
-        "Framework :: Plone :: 5.0",
-        "Framework :: Plone :: 5.1",
-        "Framework :: Plone :: 5.2",
```
Comment (gforcada) [inline setup.py:25]: Should this be kept? And add 6.0? 🤔

**Review by gforcada — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (mauritsvanrees) [reply setup.py:25]: Oops, did I remove them all?  I meant to replace them all with 6.0 only.  Fixed now.
I don't want to keep 5.2 support.

**Comment by mauritsvanrees** [comment]
BTW, I created a few PRs like this yesterday, but it is not my intention to include them in the upcoming Plone 6.0.2.  I am in no hurry to get these into releases.

**Pull request merged**